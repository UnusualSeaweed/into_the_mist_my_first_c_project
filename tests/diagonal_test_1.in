place S19
place A1
place R18
place B2
place Q17
place C3
place P16
place D4
place O15
