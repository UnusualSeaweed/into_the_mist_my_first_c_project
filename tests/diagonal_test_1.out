Black wins!
S19A1R18B2Q17C3P16D4O15
Thank you for playing!
